src/Arcus.Templates.ServiceBus.Queue/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM mcr.microsoft.com/dotnet/sdk:8.0-alpine AS base
+FROM mcr.microsoft.com/dotnet/runtime:8.0.4-jammy-chiseled AS base
```

Comment:
Just wondering why the runtime image is a 'jammy' image (Ubuntu 22.04) while the base image that we use for the build phase, is a bookworm (Debian based) image ?
https://github.com/dotnet/dotnet-docker/blob/main/README.runtime.md#linux-amd64-tags

Reply:
Ok, changed all to `jammy` in places where we use `chiseled`.